app/models/library_entry.rb

```diff
@@ -104,6 +104,14 @@ def activity
     MediaActivityService.new(self)
   end
 
+  def sync_to_mal?
+    # is it better to take present? out and pass
+    # this variable into the worker also?
```

Comment:
No, it makes more sense to handle it here, so it won't create a bunch of async jobs which do nothing